### DISCUSSION TITLE
2023년 07월 10일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#68] [General]
# 2023년 07월 10일

[ 2023-07-04 ] [ 2023-07-05 ] [ 2023-07-06 ] [ 2023-07-08 ] [ 2023-07-09 ] [ 2023-07-10 ] [ 2023-07-11 ]

https://vero.wiki/2023-07-10/

**Comment by kevstevie** [comment]
와 거기 개맛잇음

**Reply by Cyma-s** [reply]
선릉 찐 맛집 ㅇㅈ

**Comment by hong-sile** [comment]
다음 회식장소는 여기다
